app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def posts_count_updated
+    users.increment!(:posts_counter)
+  end
```

Comment:
### OPTIONAL

- [ ] You can use [rails active callbacks.](https://guides.rubyonrails.org/active_record_callbacks.html)

``` 
after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```
`after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private